misk-core/build.gradle

```diff
@@ -15,6 +15,7 @@ dependencies {
   compile dep.okio
   compile dep.okHttp
   compile dep.slf4jApi
+  compile dep.snappy
```

Comment:
I’d love to figure out a way to do this outside of misk-core. I’d like for that module to get smaller, not bigger

Reply:
Do we have a tiny utility module yet?